CollAction/Startup.cs

```diff
@@ -201,7 +202,8 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
                                                    .Sources.AddRange(Configuration["CspMediaSrc"]?.Split(";") ?? new string[0]); // Only allow self-hosted videos, or configured sources
                            cspBuilder.AddFrameAncestors().None(); // No framing allowed here (put us inside a frame tag)
                            cspBuilder.AddScriptSrc() // Only allow scripts from our own site, the aspnetcdn site, app insights and google analytics
-                                     .Self()
+                                     .Self()                                    
+                                     .UnsafeEval()  // Needed for client side validation scripts
```

Comment:
Is there no different way to solve this? Not having UnsafeEval does protect us against a whole slew of vulnerabilities that we could get if the rich text editor is in some way "leaky" (XSS), it's important for the whole defense-in-depth thing.

We could whitelist the validations if you have a list for example.

Reply:
I'll take a look. I'm not really happy with how this works either; the validators aren't really part of the typescript react code but end up in the bundle non-the-less. I'll see if I can find another solution. Probably vanilla javascript.

Reply:
The UnsafeEval is needed when the Webpack bundle is built in development mode, which is what the docker image and travis do. I'll change it so it always builds in production mode. That makes the bundle a lot smaller too.